src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -370,4 +374,27 @@ private void changeLabels() {
             ui.triggerEvent(new ShowLabelPickerEvent(getSelectedIssue().get()));
         }
     }
+
+    /**
+     * Adds a style class to the listview which changes its background to contain a loading spinning gif
+     */
+    @Override
+    protected void addPanelLoadingIndication() {
+        logger.info("Preparing to add panel loading indication");
+        listView.getStyleClass().add("listview-loading");
+
+        // Remove the items in the issue list because they are not relevant to the filter anymore.
+        // This also makes the listview background visible, since we intend to show a loading indicator on the
+        // background.
+        listView.setItems(null);
+    }
```

Comment:
So if there are for example 2 repos in the filter and we're opening a 3rd repo, from what I understand, this will cause the whole panel to go blank? I'm not sure if that's the desirable behaviour.